src/main/java/org/datadog/jenkins/plugins/datadog/DatadogUtilities.java

```diff
@@ -143,6 +147,13 @@ private static String getOS() {
                         Set<String> tagValues = tags.containsKey(tagName) ? tags.get(tagName) : new HashSet<String>();
                         tagValues.add(tagValue.toLowerCase());
                         tags.put(tagName, tagValues);
+                    } else if(tagItem.length == 1) {
```

Comment:
we should also consider the fact that there could me more than 1 colon in the tags. To do so, let's simply do `split(":", 2)`